index.js

```diff
@@ -820,8 +829,11 @@ class Encore {
      *              by individual polyfills. Using it with 'usage' will try to
      *              automatically detect which polyfills are needed for each file and
      *              add them accordingly.
+     *              Cannot be used if you have an external Babel configuration (a .babelrc
+     *              file for instance). In this case you can set the option directly into
+     *              that configuration file.
      *
-     * @param {function} callback
+     * @param {function|false} callback
```

Comment:
I suggest documenting as `function|null` rather than `function|false`. Using a boolean to represent the absence of value does not make much sense.

Reply:
btw, the error message even says `Argument 1 to configureBabel() must be a callback function or null.`

Reply:
Yeah, I initially went with `null` but for some reasons switched to `false` (and didn't update that message).

I think my reasoning was something like "that parameter is disabled and won't be used", but now that you point it out I agree that it makes more sense to put `null`.